e2e/specs/stateless/extendNames.spec.ts

```diff
@@ -310,3 +318,66 @@ test('should not show extend button on unwrapped subnames', async ({
 
   await expect(profilePage.getExtendButton).toHaveCount(0)
 })
+
+test('should be able to extend a name by a month', async ({
+  page,
+  login,
+  makePageObject,
+  makeName,
+}) => {
+  const name = await makeName({
+    label: 'legacy',
+    type: 'legacy',
+    owner: 'user',
+  })
+
+  const extendNamesModal = makePageObject('ExtendNamesModal')
+  const profilePage = makePageObject('ProfilePage')
+
+  await profilePage.goto(name)
+  await login.connect()
+
+  await page.pause()
+  const timestamp = await profilePage.getExpiryTimestamp()
+  await profilePage.getExtendButton.click()
+
+  await test.step('should be able to pick by date', async () => {
+    const dateSelection = page.getByTestId('date-selection')
+    await expect(dateSelection).toHaveText('Pick by date')
+
+    await dateSelection.click()
+  })
+
+  await test.step('should set and render a date properly', async () => {
+    const expiryTime = (await profilePage.getExpiryTimestamp()) / 1000
+    const calendar = page.getByTestId('calendar')
+    const monthLater = await page.evaluate(
+      (ts) => {
+        return new Date(ts)
+      },
+      (expiryTime + daysToSeconds(30)) * 1000,
+    )
+
+    await calendar.fill(dateToDateInput(monthLater))
+    await expect(page.getByTestId('calendar-date')).toHaveValue(
+      secondsToDateInput(expiryTime + roundDurationWithDay(monthLater, expiryTime)),
```

Comment:
test passes locally but fails in CI. has something to do with time manipulation i suppose